Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -2106,6 +2106,7 @@ if (window.testRunner) {
     testRunner.setAllowedMenuActions = actions => post(['SetAllowedMenuActions', actions]);
     testRunner.setOpenPanelFiles = files => post(['SetOpenPanelFileURLs', files]);
     testRunner.setOpenPanelFilesMediaIcon = iconBytes => post(['SetOpenPanelFileURLsMediaIcon', iconBytes]);
+    testRunner.setPrinting = value => post(['SetPrinting', value]);
```

Comment:
This sends a non-synchronous message, but it is read by a synchronous message.  In WebKit, synchronous messages are sometimes delivered before asynchronous messages, so this introduces a race condition.  I think we'll need to stay with the idl for now and send a synchronous message to set it to ensure that its setting is received by the UI process before its reading.